packages/core/src/transports/base.ts

```diff
@@ -12,4 +12,8 @@ export abstract class BaseTransport implements Transport {
   get agent(): Agent {
     return agent;
   }
+
+  shutdown(): void {
```

Comment:
you can make this one abstract since it will "force" transports to implement it.

Reply:
I know, but I thought most transports don't need to, so made it optional